docs/how-to/document-search/use_rephraser.md

```diff
@@ -0,0 +1,41 @@
+# How-To: Use Rephraser
+`ragbits-document-search` contains a `QueryRephraser` module that could be used for creating an additional query that
+improves the original user query (fixes typos, handles abbreviations etc.). Those two queries are then sent to the document search
+module that can use them to find better matches.
+
+This guide will show you how to use `QueryRephraser` and how to create your custom implementation.
+
+## LLM rephraser usage
+To use a rephraser you need to provide it during `DocumentSearch` construction. In the following example we will use
+`LLMQueryRephraser` and default `QueryRephraserPrompt`.
+```python
+from ragbits.core.llms.litellm import LiteLLM
+from ragbits.document_search.retrieval.rephrasers.llm import LLMQueryRephraser
+from ragbits.document_search.retrieval.rephrasers.prompts import QueryRephraserPrompt
+
+document_search = DocumentSearch(
+    query_rephraser=LLMQueryRephraser(LiteLLM("gpt-3.5-turbo"), QueryRephraserPrompt),
+    ...
+)
+results = await document_search.search("<query>")
```

Comment:
maybe we should show how to use rephraser independently too ? if it is not intended to be used directly perhaps we should not include such components in public api